tests/kafkatest/tests/core/transactions_test.py

```diff
@@ -53,7 +53,11 @@ def __init__(self, test_context):
         self.zk = ZookeeperService(test_context, num_nodes=1)
         self.kafka = KafkaService(test_context,
                                   num_nodes=self.num_brokers,
-                                  zk=self.zk)
+                                  zk=self.zk,
+                                  # Reducing timeout of transaction can quickly cleanup the unstable offsets.
+                                  # IN hard_bounce mode, transaction is broke ungracefully. Hence, it produces unstable
+                                  # offsets which obstructs TransactionalMessageCopier from receiving position of group.
+                                  transaction_timeout=2000)
```

Comment:
The default transaction.abort.timed.out.transaction.cleanup.interval.ms is 10 secs. In the jira, the consumer seems to have failed without making progress for 30 secs. So, will reducing this timeout help?

Reply:
@junrao thanks for your great reviews!!! This approach is not correct. The "number" which should be reduced is "transaction timeout" rather than "cleanup interval".